src/build.rs

```diff
@@ -96,11 +96,23 @@ pub(crate) fn build_impl(
 
     let kernel = build_kernel(&out_dir, &bin_name, &args, verbose)?;
 
+    let maybe_package = if let Some(ref path) = config.package_filepath {
+        Some(File::open(path).context("Unable to open specified package file")?)
```

Comment:
Could you use `with_context` instead of `context` here to also show the underlying error.